content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -0,0 +1,431 @@
+{{< /table >}}
+
+{{< note >}}
+Assim que um _feature gate_ atinge a disponibilidade geral, ele é removido desta
+lista e o seu valor fica bloqueado em `true` por padrão. Ou seja, a funcionalidade
+estará sempre ativa.
+{{< /note >}}
+
+Descrição dos _feature gates_:
+
+`PublicKeysECDSA`
+: Pode ser utilizado para criar um cluster que utilize certificados ECDSA no
+lugar do algoritmo RSA padrão. A renovação dos certificados ECDSA existentes
+também é suportada utilizando o comando `kubeadm certs renew`, mas você não pode
+alternar entre os algoritmos RSA e ECDSA dinamicamente ou durante atualizações.
+
+`RootlessControlPlane`
+: Quando habilitada esta opção, os componentes da camada de gerenciamento cuja
+instalação de Pods estáticos é controlada pelo kubeadm, como o `kube-apiserver`,
+`kube-controller-manager`, `kube-scheduler` e `etcd`, têm seus contêineres
+configurados para rodarem como usuários não-root. Se a opção não for habilitada,
+estes componentes são executados como root. Você pode alterar o valor deste
+_feature gate_ antes de atualizar seu cluster para uma versão mais recente do
+Kubernetes.
+
+`UnversionedKubeletConfigMap`
+: Esta opção controla o nome do {{< glossary_tooltip text="ConfigMap" term_id="configmap" >}}
+onde o kubeadm armazena os dados de configuração do kubelet. Quando esta opção
+não for especificada ou estiver especificada com o valor `true`, o ConfigMap
+será nomeado `kubelet-config`. Caso esteja especificada com o valor `false`, o
+nome do ConfigMap incluirá as versões maior e menor do Kubernetes instalado
+(por exemplo, `kubelet-config-{{< skew currentVersion >}}`). O kubeadm garante
+que as regras de RBAC para leitura e escrita deste ConfigMap serão apropriadas
+para o valor escolhido. Quando o kubeadm cria este ConfigMap (durante a execução
+dos comandos `kubeadm init` ou `kubeadm upgrade apply`), o kubeadm irá respeitar
+o valor da opção `UnversionedKubeletConfigMap`. Quando tal ConfigMap for lido
+(durante a execução dos comandos `kubeadm join`, `kubeadm reset`,
+`kubeadm upgrade...`), o kubeadm tentará utilizar o nome do ConfigMap sem a
+versão primeiro. Se esta operação não for bem-sucedida, então o kubeadm irá
+utilizar o nome legado (versionado) para este ConfigMap.
+
+{{< note >}}
+Informar a opção `UnversionedKubeletConfigMap` com o valor `false` é suportado,
+mas está **descontinuado**.
+{{< /note >}}
+
+### Adicionando parâmetros do kube-proxy {#kube-proxy}
+
+Para informações sobre como utilizar parâmetros do kube-proxy na configuração
+do kubeadm, veja:
+- [referência do kube-proxy](/docs/reference/config-api/kube-proxy-config.v1alpha1/)
+
+Para informações sobre como habilitar o modo IPVS com o kubeadm, veja:
+- [IPVS](https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/ipvs/README.md)
+
+### Informando opções personalizadas em componentes da camada de gerenciamento {#control-plane-flags}
+
+Para informações sobre como passar opções aos componentes da camada de gerenciamento,
```

Comment:
I added "as" and liked. Think about that "como passar as opções aos comandos"

Reply:
Fixed!